modules/contact/src/actions/ContactAction.C

```diff
@@ -684,9 +709,6 @@ ContactAction::commonParameters()
 {
   InputParameters params = emptyInputParameters();
 
-  MooseEnum orders(AddVariableAction::getNonlinearVariableOrders());
```

Comment:
I think we shouldn't just delete this parameter -- this will make a bunch of cases that have that set break.  I'd say we should make it a deprecated parameter and generate an error if the order specified by the user is different from what you get with your automatic stuff.

Reply:
If the setting is not what is generated by the automatic stuff then it is just wrong and will eventually lead to a segfault. That's what this fix is trying to prevent. I think having `addDeprecatedParam ("order", orders, "The finite element order", "This parameter is no longer used, simply remove it")` or something like that is fine but then we do not have an getParam("order"). So the code will still parse that it is set, but not use it since it isn't needed.  Also in terms of breaking stuff, there was one BISON test for which I patched earlier today. But there could be external users that would hit this issue.

Reply:
We realized that adding this as a deprecated parameter will generate warning messages for the cases where people set 'order' in their GlobalParams, which isn't good either. Because of that, it's best to just remove it. At worst, people with existing models will just need to delete that parameter.